### PR TITLE
BDD100K : Renaming dataset classname

### PR DESCRIPTION
**Pull request by ananno**
Changing classname `BDD100K` to `BDD100KObjectDetection`

## Motivation and Context

BDD100K will have two dataset classes : ObjectDetection and Segmentation. So original dataset classname must be renamed accordingly.

## Description

Current BDD100K dataset file has only one dataset class `BDD100K` which is for object detection task. In order to add two separate datasets for object detection and segmentation the classname `BDD100K` must be modified accordingly.

## How has this been tested?

`tox -e pytest`
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature / Optimization (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by ananno** [comment]
@iizukak San:
I began to separate #399 into multiple smaller PRs. This is one of them. Please review.

File: lmnet/lmnet/datasets/bdd100k.py [hunk 1]
```diff
@@ -1,16 +1,15 @@
 # -*- coding: utf-8 -*-
-import functools
-import glob
 import json
-from os.path import basename, join, splitext
-
+import glob
+import functools
 import numpy as np
+from os.path import basename, join, splitext
```
Comment (iizukak) [inline lmnet/lmnet/datasets/bdd100k.py]: Why did you change import order?

**Comment by iizukak** [comment]
@ananno Thank your for make PR smaller.
We can review and merge your PR step by step.

Can you assign reviewer on Github both ownership and readability?

File: lmnet/lmnet/datasets/bdd100k.py (hunk 1 above)
Comment (ananno) [reply lmnet/lmnet/datasets/bdd100k.py]: So that it visually looks good. Is there a problem with order?

**Comment by ananno** [comment]
> @ananno Thank your for make PR smaller.
> We can review and merge your PR step by step.
> 
> Can you assign reviewer on Github both ownership and readability?

@iizukak san:
I’m sorry I don’t have permission to do that. I’ve tried when I created the PRs.

File: lmnet/lmnet/datasets/bdd100k.py (hunk 1 above)
Comment (tkng) [reply lmnet/lmnet/datasets/bdd100k.py]: (Sorry for duplicated comment) @ananno Could you apply `isort` to sort import orders?

Since aesthetic standards vary from person to person, it should not be applied as it is. We should follow some rules, and in the case of Python, I think `isort` is a safe standard.

Comment (ananno) [reply lmnet/lmnet/datasets/bdd100k.py]: @tkng Agreed. I've applied `isort` and will use it from now on.

**Comment by ananno** [comment]
@tkng, @iizukak San:
I've applied isort as requested. Please review one more time.
